tests/s3/test_object_workflow_operations.py

```diff
@@ -107,6 +111,43 @@ def create_bucket_put_objects(self, bucket_name, object_count):
 
         return obj_list
 
+    def create_bucket_put_get_object(self, bucket_name, obj_key_lst, file_path, mb_count):
+        """
+        Function creates a bucket and uploads an object
+        and then retrieves object from specified S3 bucket.
+
+        :param bucket_name: Name of bucket to be created.
+        :param obj_key_lst: List of object keys.
+        :param file_path: Path of the file to be created and uploaded to bucket.
+        :param mb_count: Size of file in MBs.
+        """
+        for key in obj_key_lst:
+            resp = self.s3_test_obj.create_bucket_put_object(
+                bucket_name, key, file_path, mb_count)
+            assert resp[0], resp[1]
+            resp = self.s3_test_obj.get_object(
+                bucket_name, key)
+            assert resp[0], resp[1]
+            self.log.info(
+                "GET operation successful for object %s from bucket %s",
+                key, bucket_name)
+            resp = self.s3_test_obj.delete_bucket(self.bucket_name, force=True)
+            assert resp[0], resp[1]
+
+    def random_strings_list(self, string_count = 5):
```

Comment:
I'm wondering if having a list of random strings will add any additional value to these tests. Maybe we can a generate single random string at the setup_method phase and then use that for generating combinations with the specific character being tested.